src/services/walletconnect/WalletConnectWallet.ts

```diff
@@ -234,10 +234,12 @@ class WalletConnectWallet {
   /**
    * Subscribe to session delete
    */
-  public onSessionDelete(handler: () => void) {
+  public onSessionDelete = (handler: (session: SessionTypes.Struct) => void) => {
```

Comment:
There were changed to arrow functions because there were otherwise complaints that `this` was `undefined`.